micronota/parsers/gff.py

```diff
@@ -0,0 +1,238 @@
+    +--------------------------------+----------------------------------+
+    | STRAND - +/-/./?               | 1 per entry                      |
+    +--------------------------------+----------------------------------+
+    | PHASE - only for TYPE="CDS"    | 1 per entry                      |
+    +--------------------------------+----------------------------------+
+    | ATTR - feature attributes (tag)| 1 per entry                      |
+    +--------------------------------+----------------------------------+
+
+
+Column 9 (ATTR) tags have predefined meanings:
+
+ID
+Indicates the unique identifier of the feature.
+IDs must be unique within the scope of the GFF file.
+
+Name
+Display name for the feature. This is the name to be displayed to the user.
+Unlike IDs, there is no requirement that the Name be unique within the file.
+
+Alias
+A secondary name for the feature.
+It is suggested that this tag be used whenever a secondary identifier for
+the feature is needed, such as locus names and accession numbers.
+Unlike ID, there is no requirement that Alias be unique within the file.
+
+Parent
+Indicates the parent of the feature.
+A parent ID can be used to group exons into transcripts, transcripts into genes
+and so forth. A feature may have multiple parents. Parent can *only* be used
+to indicate a partof relationship.
+
+Target
+Indicates the target of a nucleotide-to-nucleotide or
+protein-to-nucleotide alignment. The format of the value is "target_id start
+end [strand]", where strand is optional and may be "+" or "-".
+If the target_id contains spaces, they must be escaped as hex escape %20.
+
+Gap
+The alignment of the feature to the target if the two are not collinear
+(e.g. contain gaps).
+The alignment format is taken from the CIGAR format described in the Exonerate
+documentation.
+
+Derives_from
+Used to disambiguate the relationship between one feature and another when the
+relationship is a temporal one rather than a purely structural "part of" one.
+This is needed for polycistronic genes.
+
+Note
+A free text note.
+
+Dbxref
+A database cross reference. See the GFF3 specification for more information.
+
+Ontology_term
+A cross reference to an ontology term.
+
+Multiple attributes of the same type are indicated by separating
+the values with the comma ","
+
+
+Format Support
+--------------
+**Has Sniffer: Yes**
+
++------+------+---------------------------------------------------------------+
+|Reader|Writer|                          Object Class                         |
++======+======+===============================================================+
+|Yes   |No    |:mod:`skbio.metadata.IntervalMetadata` objects                 |
++------+------+---------------------------------------------------------------+
+|Yes   |No    |generator of :mod:`skbio.metadata.IntervalMetadata` objects    |
++------+------+---------------------------------------------------------------+
+
```

Comment:
would you mind provide writer as well? write IntervalMetadata object into GFF3. it will be very useful.